task5/src/main/java/com/clouway/core/Transaction.java

```diff
@@ -0,0 +1,20 @@
+package com.clouway.core;
+
+/**
+ * Created by Kristiyan Petkov  <kristiqn.l.petkov@gmail.com> on 14.06.16.
+ */
+public class Transaction {
+  public int ID;
```

Comment:
`Ingeger` instead of `int` ... same with Doubles . They are objects and give you flexibility